pebblo/app/service/loader/loader_doc_service.py

```diff
@@ -310,7 +310,10 @@ def process_request(self, data):
                 )
                 loader_response_output.append(doc_obj)
 
-            if self.data["loading_end"]:
+            # Commit before generating report, so that even if report generation fails then values would be in db.
+            self.db.session.commit()
+
+            if self.data.get("loading_end", False):
```

Comment:
no need to do False even if None return False

Reply:
We always get `loading_end` either `True` or `False`

Reply:
Fixed.